### PR TITLE
Create standard picture

### PR DESCRIPTION
**Pull request by ArneSchulze**

### CODE REVIEW

File: app/src/main/java/de/ameyering/wgplaner/wgplaner/section/home/ItemDetailActivity.java [hunk 1]
```diff
@@ -149,41 +149,41 @@ private void initActionView() {
...
-                                    addPriceActionView.setEnabled(true);
-                                });
-                            }
-                        }
+                new ServerCallsInterface.OnAsyncCallListener<ListItem>() {
+                    @Override
+                    public void onFailure(ApiException e) {
+                        if (e == null) {
+                            runOnUiThread(() -> {
+                                itemPriceEditLayout.setError(getString(R.string.dialog_add_price_error));
```
Comment (bugwelle) [inline app/src/main/java/de/ameyering/wgplaner/wgplaner/section/home/ItemDetailActivity.java]: Can you move the null condition inside to aboid duplicated code (`setEnabled`, etc)?

File: app/src/main/java/de/ameyering/wgplaner/wgplaner/section/settings/GroupSettingsActivity.java [hunk 2]
```diff
@@ -426,4 +459,34 @@ protected void onResume() {
         dataProvider.addOnDataChangeListener(groupListener);
         super.onResume();
     }
+
+    private Bitmap createStandardBitmap(String displayName) {
```
Comment (bugwelle) [inline app/src/main/java/de/ameyering/wgplaner/wgplaner/section/settings/GroupSettingsActivity.java]: This whole method is duplicated.

File: app/src/main/java/de/ameyering/wgplaner/wgplaner/section/settings/ProfileSettingsActivity.java [hunk 3]
```diff
@@ -132,6 +157,36 @@ public void onSuccess(SuccessResponse result) {
         });
     }
 
+    private Bitmap createStandardBitmap(String displayName) {
+        Bitmap standard = Bitmap.createBitmap(standard_width, standard_width, Bitmap.Config.ARGB_8888);
```
Comment (bugwelle) [inline app/src/main/java/de/ameyering/wgplaner/wgplaner/section/settings/ProfileSettingsActivity.java]: Three times the same method?

File: app/src/main/java/de/ameyering/wgplaner/wgplaner/section/setup/fragment/CreateGroupFragment.java [hunk 4]
```diff
@@ -210,4 +209,34 @@ private boolean checkInputsAndReturn() {
 
         return true;
     }
+
+    private Bitmap createStandardBitmap(String displayName) {
+        Bitmap standard = Bitmap.createBitmap(standard_width, standard_width, Bitmap.Config.ARGB_8888);
+        Canvas canvas = new Canvas(standard);
```
Comment (bugwelle) [inline app/src/main/java/de/ameyering/wgplaner/wgplaner/section/setup/fragment/CreateGroupFragment.java]: Oh my.. 4 times?

File: app/src/main/java/de/ameyering/wgplaner/wgplaner/section/settings/GroupSettingsActivity.java (hunk 2 above)
Comment (ArneSchulze) [reply app/src/main/java/de/ameyering/wgplaner/wgplaner/section/settings/GroupSettingsActivity.java]: That's quick and dirty... I will introduce better standard image handling in another branch. Now it's too much work to do

**Comment by bugwelle** [comment]
We can ignore this quick and dirty copy&paste for now. Time issues I'd say...

**Pull request merged**